src/main/java/org/folio/marc4ld/service/marc2ld/field/property/transformer/SeparatePropertyTransformer.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.marc4ld.service.marc2ld.field.property.transformer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SeparatePropertyTransformer implements PropertyTransformer {
+
+  @Override
+  public Collection<Map<String, List<String>>> apply(Map<String, List<String>> properties) {
```

Comment:
Hi @aleksei-pronichev,

I think this logic will cause issues for [marc field 651](https://www.loc.gov/marc/bibliographic/bd651.html) which is mapped to a Place resource.

In 651, the place name ($a) is non repeatable. However, Miscellaneous Info ($g) is repeatable. In such case, we will throw an exception in line 24.

Reply:
add some test cases for
-651 (conflict)
-650 Place
-650 Form